src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+public class ParserTest {
+    @Test
+    public void createTaskTestTodoException() {
```

Comment:
Would it be better to rename the test methods following the test methods conventions?

```
featureUnderTest_testScenario_expectedBehavior()
```